### ISSUE TITLE
v5.0.0 does not work in tinyhttp

### ISSUE THREAD

**Issue by dwbboy** [tinyhttp/milliparsec#23]
index.js
```
import { App } from '@tinyhttp/app'
import { json } from 'milliparsec'

const app = new App()

app
  .use(json())
  .post('/', (req, res) => {
    res.json(req.body)
  })
  .listen(3000)
```
package.json
```
{
  "name": "tt",
  "module": "index.js",
  "type": "module",
  "scripts": {
    "start": "node index.js"
  }
  "dependencies": {
    "@tinyhttp/app": "^2.4.0",
    "milliparsec": "^5.0.0"
  }
}
```

**Comment by dwbboy** [comment]
```
curl.exe localhost:3000 -d '{"some": "data"}'
JSON Parse error: Expected '}'
```

**Comment by talentlessguy** [comment]
what exactly doesn't work? like the body doesn't get parsed?

**Comment by talentlessguy** [comment]
fixed in 5.0.1
